.config/dotnet-tools.json

```diff
@@ -20,11 +20,11 @@
         "dotnet-serve"
       ]
     },
-    "fantomas-tool": {
-      "version": "5.0.0-alpha-002",
+    "fantomas": {
+      "version": "5.0.0-alpha-010",
```

Comment:
Ah shoot, this is already merged. @yisusalanpng I released alpha 11 on Friday. Would you mind making another PR?